lib/commands/connect/mapping-reload.js

```diff
@@ -17,16 +17,14 @@ module.exports = {
   needsApp: true,
   needsAuth: true,
   run: cli.command(co.wrap(function* (context, heroku) {
-    let connection = yield api.withConnection(context, heroku);
-    let mapping = yield api.withMapping(connection, context.args.mapping);
+    yield cli.action('reloading mapping', co(function* () {
+      let connection = yield api.withConnection(context, heroku);
+      let mapping = yield api.withMapping(connection, context.args.mapping);
 
-    let response = yield api.request(context.auth.password, 'POST', '/api/v3/mappings/' + mapping.id + '/actions/reload');
-    if (response.statusCode === 202) {
-      cli.log('Initated reload of ' + mapping.object_name);
-    }
-    else {
-      cli.log('Received unexpected resonse during mapping reload ...');
-      api.printResponseError(response);
-    }
+      let response = yield api.request(context.auth.password, 'POST', '/api/v3/mappings/' + mapping.id + '/actions/reload');
+      if (response.statusCode !== 202) {
+        throw new Error(response.json.message || 'unknown error');
+      }
```

Comment:
Like above, it'll automatically look like this:

```
reloading mapping... done
```

Though now that I look at that, it's a bit misleading, because the reload isn't complete. I'll update it to say "initiating reload..." instead.